tests/person_spec.rb

```diff
@@ -0,0 +1,20 @@
+require '../classes/person'
+
+describe 'Test the Person class:' do
+  it 'Check patameters of an instance' do
+    person = Person.new(12, name: 'Sandy')
+    expect(person.age).to be 12
+    expect(person.name).to eql 'Sandy'
+    expect(person.parent_permission).to eql true
+  end
+
+  it 'Expect the can_use_services methode to be true:' do
+    person = Person.new(12, name: 'Sandy')
+    expect(person.can_use_services?).to eql true
+  end
+
+  it 'Expect the can_use_services methode to be false:' do
+    person = Person.new(17, name: 'Sandy', parent_permission: false)
+    expect(person.can_use_services?).to eql false
+  end
```

Comment:
- [ ] Good job with test coverage, but since we are aiming for 100% coverage, it will be nice to cover all our methods in all our classes with tests, for example, in this `Person class`, the following methods still needs to be covered with tests: 
- `When validating name` (method for validating name)
- `When adding a new rentals` (method for adding a new rental)

### Please make all your methods in all your classes are covered with tests.